plugins/Marketplace/config/test.php

```diff
@@ -83,11 +83,37 @@
 
         $service->authenticate($accessToken);
 
-        function removeReviewsUrl($content)
+        function updatePluginUrlsForTests(&$plugin)
```

Comment:
It's quite hard to understand what this code is doing, would it be possible to name the function with something that better represents what its doing?

Reply:
Whatever you think makes sense here so that the person coming to look at this code next isn't confused.

Reply:
Updated method names and added inline comments to better explain what they do.